assets/js/googlesitekit/datastore/site/index.js

```diff
@@ -26,64 +26,20 @@ import reset from './reset';
 import { STORE_NAME } from './constants';
 import notifications from './notifications';
 
-export const INITIAL_STATE = Data.collectState(
-	connection.INITIAL_STATE,
-	info.INITIAL_STATE,
-	reset.INITIAL_STATE,
-	notifications.INITIAL_STATE,
-);
-
 export { STORE_NAME };
 
-export const actions = Data.addInitializeAction(
-	Data.collectActions(
-		Data.commonActions,
-		connection.actions,
-		info.actions,
-		reset.actions,
-		notifications.actions,
-	)
-);
-
-export const controls = Data.collectControls(
-	Data.commonControls,
-	connection.controls,
-	info.controls,
-	reset.controls,
-	notifications.controls,
-);
-
-export const reducer = Data.addInitializeReducer(
-	INITIAL_STATE,
-	Data.collectReducers(
-		connection.reducer,
-		info.reducer,
-		reset.reducer,
-		notifications.reducer,
-	)
+const store = Data.combineStores(
+	{
+		actions: Data.addInitializeAction( Data.commonActions ),
+		controls: Data.commonControls,
+		reducer: Data.addInitializeReducer( {}, ( state ) => ( { ...state } ) ),
+	},
```

Comment:
@tofumatt should this maybe be exposed as `Data.commonStore` since all stores will need this?

Reply:
Yeah, I think that makes sense actually 👍